GocadRelatedScripts/createGOCADTSurfNXNY_netcdf.py

```diff
@@ -114,14 +114,12 @@
 nodes=np.zeros((nnodes+1,3))
 triangles=np.zeros((ntriangles,3))
 
-k=1
-for j in range(0,NY):
-    for i in range(0,NX):
-       if args.proj!='':
-          nodes[k,:] = pyproj.transform(lla, myproj, lon[i], lat[j], 1e3*elevation[j,i], radians=False)
-       else:
-          nodes[k,:]=  [lon[i], lat[j], elevation[j,i]]
-       k=k+1
+xv, yv = np.meshgrid(lon, lat)
+nodes[1:,0]=xv.flatten()
```

Comment:
Why do we leave nodes[0,:] untouched? Is there a reason, why we keep the first node as zero?

Reply:
that s because ts file usually start at vertex 1 and not 0. But actually I tested with 0 and it works so I changed that

Reply:
Ok